Readme.md

```diff
@@ -42,42 +96,56 @@ Format the given value in bytes into a string. If the value is negative, it is k
 **Example**
 
 ```js
-bytes(1024);
+bytes(1000);
```

Comment:
Is this a typo in the documentation? How can `bytes(1000)` output `1kB` and `1000B` at the same time?